packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -158,9 +160,10 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     }
 
     switch (imageSource) {
-      case SOURCE_CAMERA:
+      case SOURCE_CAMERA: {
         [self checkCameraAuthorization];
         break;
+      }
```

Comment:
Why this change?

Reply:
I am not sure why I changed this. I reverted it.

Reply:
I am not sure why I changed this. I reverted it.